### PR TITLE
Additional user fetch if github is empty

### PR DESCRIPTION
**Pull request by Sayan-**
quick fix for an edge case where we don't receive a fully populated whoswho.User

TODO -> update tests

### CODE REVIEW

File: bot.go [hunk 1]
```diff
@@ -392,15 +392,21 @@ func (bot *Bot) pickIndividual(ev *slack.MessageEvent, individualSlackID string,
...
-			"event-text":           ev.Text,
-			"user-pickabot-config": user.Pickabot,
-			"user-slack":           user.Slack,
-			"user-slack-id":        user.SlackID,
-		})
-		return fmt.Errorf("No github account for slack user <@%s>", user.SlackID)
+		// try to fetch the user from SlackID
+		user, err = bot.WhoIsWhoClient.UserBySlackID(user.SlackID)
+		// error if we can't get an update or if there is still no valid account associated
+		if user.Github == "" || err != nil {
```
Comment (nathanleiby) [inline bot.go]: nit: i might handle the error separately, so you can log the actual error that happened

Comment (Sayan-) [reply bot.go]: Ah ya - good call https://github.com/Clever/pickabot/pull/14/commits/e1f1e737390ebe9aec4792ccf4d590b1fa355e17
